hack/dev-push-hyperkube.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script will build and push the hyperkube image.
+# This is useful for rapidly testing changes in a cluster that is
+# powered by hyperkube. You can configure the cluster to use a
```

Comment:
@davidopp I've already updated the PR for the Azure docs to speak to "AZURE_HYPERKUBE_SPEC" which is how someone can use a specific hyperkube image for the cluster. Given that, I think the comment should be adjusted to something like:

"This script will build the hyperkube image and then push to to the repository referred to by KUBE_DOCKER_REGISTRY and KUBE_DOCKER_OWNER, and tagged with KUBE_DOCKER_VERSION."

The remainder of the comment is specific to Azure (or any other cluster that is bootstrapped with hyperkube).

That sound okay?